Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -43,6 +32,17 @@
 using Ninject.Activation;
 using Ninject.Web.Common;
 using Owin;
+using System;
```

Comment:
Kudu is using the VS default namespace ordering which places System before others. e.g. see http://stackoverflow.com/questions/14829853/vs2012-sort-usings-doesnt-move-system-usings-first